amazon-msk/cdk/lib/SecurePublicAccessStack.ts

```diff
@@ -364,6 +385,47 @@ export class SecurePublicAccessStack extends cdk.Stack {
 
     autoScalingGroup.attachToNetworkTargetGroup(targetGroup);
 
+    const metricsNamespace = context.cloudwatch.metrics?.namespace;
+    const interval = context.cloudwatch.metrics?.interval;
+    if (metricsNamespace && interval) {
+
+      const metricWorkerUtilization = new cw.Metric({
+        namespace: metricsNamespace,
+        metricName: 'engine.worker.utilization',
+        statistic: 'Average',
+        period: cdk.Duration.seconds(interval),
+      });
```

Comment:
```suggestion
    if (context.cloudwatch.metrics) {

      const metricWorkerUtilization = new cw.Metric({
        namespace: context.cloudwatch.metrics.namespace,
        metricName: 'engine.worker.utilization',
        statistic: 'Average',
        period: cdk.Duration.seconds(context.cloudwatch.metrics.interval),
      });
```

`namespace` is required on `metrics` and `interval` is defaulted as necessary if `metrics` is present.